src/protocolgame.cpp

```diff
@@ -2564,6 +2565,16 @@ void ProtocolGame::sendSaleItemList(const std::vector<ShopInfo>& shop, const std
 	writeToOutputBuffer(playermsg);
 
 	playermsg.reset();
+	#if CLIENT_VERSION >= 1100
+	playermsg.addByte(0xEE);
+	playermsg.addByte(0x00);
+	playermsg.add<uint64_t>(player->getBankBalance());
+	playermsg.addByte(0xEE);
+	playermsg.addByte(0x01);
+	playermsg.add<uint64_t>(playerMoney);
+	playermsg.addByte(0x7B);
+	playermsg.add<uint64_t>(playerMoney);
+	#elif CLIENT_VERSION >= 973
```

Comment:
Can we please remove the client versions and keep only the most recent?
Also, 0xEE is sendBalance and its a message itself, cant we isolate this in a function like sendBalance(type, value) and call it twice for bank and money?
Or at least use .reset() between the different messages to make it easier to read.